src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -30,10 +30,13 @@ public static void ConcurrentInitializeTest()
                 {
                     countdownEvent.Signal();
                     countdownEvent.Wait(ThreadTestHelpers.UnexpectedTimeoutMilliseconds);
-                    Assert.True(ThreadPool.SetMinThreads(processorCount, processorCount));
                 };
 
+            // ThreadPool.SetMinThreads returns false on UapAot
```

Comment:
I think it's still useful to allow setting min/max number of threads (sometimes the scenario favors running more threads than the processor count to allow short work to run and complete while relatively long CPU-bound work is holding up some threads). But it probably isn't a good idea to hook those into SetMinThreads / SetMaxThreads since they are extensively used to work around the issues of the CoreCLR thread pool, and the Windows thread pool would not have many of those issues. Perhaps it can be exposed later as a config option or a different API or something.